docs/learn/court.md

```diff
@@ -6,8 +6,298 @@ title: Decentralized Court
+the staked amount of each participant. This comes in handy in order to use
+binary searches for items inside the court pool. Court participants can increase
+their stake at any time by calling `join_court`or `delegate` with a higher
+amount than the previous call to these functions. To decrease the stake, it’s
+necessary to exit the court (see ”Exiting court”). To update the pool item
+associated to a court participant, there are two binary searches, the first for
+finding the pool item (the search key is a tuple of stake and the participant
+account id) and with the previous data and the second for inserting the updated
+data inside the pool again.
+
+For the `delegate` extrinsic, the function argument `delegations` contains a
+list of account ids. The dispatch function ensures that the list of
+`delegations` contains actually account ids which are actively participating
+jurors. The list needs to contain at least one account and all account ids need
+to be unique.
+
+### Exiting the Court
+
+Each juror and delegator can exit the court system to retrieve their remaining
+funds. If the juror or delegator is still actively involved in inner court
+cases, only the unused (non-active) funds are returned. The unused stake is
+equivalent to the stake that was not already selected by the selection algorithm
+(see “The selection algorithm”). In order to exit the court and get the funds
+back as a juror or delegator, one has to call the `prepare_exit_court`
+extrinsic. This extrinsic removes the participant from the court pool and saves
+the current block number to notice when the unused funds can be returned.
+
+With the introduction of the inflation period it is required to restrict court
+hopping. It is possible that users enter the court whenever the rewards of the
+inflationary system get spend (see chapter “Incentives”). Thus, we put a locking
+period of `InflationPeriod` in place in the case that a juror wants to exit the
+court. So after a participant requested to leave the court system (extrinsic
+`prepare_exit_court`), the participant has to wait at least another inflation
+period to get the staked funds back. To finally return the unused funds, the
+participant needs to evoke the `exit_court` dispatch function. The used funds
+(`active_lock`) still remain locked.
+
+### Calculating necessary Draw Weights
+
+If a juror votes against the plurality decision, they are penalized by a
+multiple of a constant amount (`n * MinJurorStake`). The penalized amount is
+rewarded proportionally to the jurors and delegators who backed the most-voted
+outcome. The amount of penalty risk depends on the juror's overall stake. When a
+court case is triggered by the prediction markets functionality, the court
+requests a small multiple of the constant amount (currently
+`31 * MinJurorStake`) from the total stake of all jurors and delegators
+randomly. The current configuration of `MinJurorStake` is `500 ZTG`. So,
```

Comment:
The formatting of amounts of currency is inconsistent throughout the document. Sometimes it's in backticks, like here, sometimes its not.

Reply:
https://github.com/zeitgeistpm/documentation/pull/81/commits/5063537295985e0db33bc808be8220ca164b2d1c